src/components/Cardsfolder/index.jsx

```diff
@@ -0,0 +1,90 @@
+import { useFetch } from "../../utils/hooks/useFetch";
+import { formatDate, generateTimeText } from "../../utils/hooks/date";
+import thumbnail from "../../assets/thumbnail.svg";
+import styles from "./index.module.css";
+import starticon from "../../assets/staricon.svg";
+import moreoptionicon from "../../assets/moreoptionicon.svg";
+import { useState } from "react";
+
+function Cardsfolder(props) {
+  const CardData = useFetch(props.url);
```

Comment:
파스칼케이스는 컴포넌트명에만 쓰는 게 일반적인 컨벤션입니다!
훅 내부의 데이터는 카멜케이스로 가져와주세요~
```suggestion
  const cardData = useFetch(props.url);
```